src/System.Collections/src/System/Collections/Generic/HashSet.cs

```diff
@@ -74,22 +74,22 @@ public class HashSet<T> : ICollection<T>, ISet<T>, IReadOnlyCollection<T>, ISeri
         private int _count;
         private int _lastIndex;
         private int _freeList;
-        private IEqualityComparer<T> _comparer = default!;
+        private IEqualityComparer<T>? _comparer = default!;
```

Comment:
Do you still need the `!`? I don't think so.

Reply:
You're right. Actually the whole `default!` isn't required. Do you think I should fix it now, or commit with other changes (if any) when the review is complete? Thanks.

Reply:
Let's do it in a separate PR 😄 -- feel free to start preparing it.